auth0/src/test/java/com/auth0/android/provider/WebAuthProviderTest.java

```diff
@@ -1324,7 +1342,8 @@ public void shouldCalculateExpiresAtDateOnResumeAuthentication() throws Exceptio
         WebAuthProvider.init(account)
                 .useCodeGrant(false)
                 .start(activity, callback, REQUEST_CODE);
-        WebAuthProvider.getInstance().setCurrentTimeInMillis(CURRENT_TIME_MS);
+        OAuthManager managerInstance = (OAuthManager) WebAuthProvider.getManagerInstance();
```

Comment:
This is the only part where I need to cast the instance to a particular manager. Because the `LogoutManager` doesn't use (and doesn't need) to handle the system Clock. Again: These managers are package internal classes.